### ISSUE TITLE
Two VB error codes have same numeric value

### ISSUE THREAD

**Issue by nguerrera** [dotnet/roslyn#13134]
https://github.com/dotnet/roslyn/blob/master/src/Compilers/VisualBasic/Portable/Errors/Errors.vb#L1689

``` VB
ERR_SourceLinkRequiresPortablePdb = 37258
ERR_TupleExplicitNamesOnAllMembersOrNone = 37258
```

@VSadov @tmat Merge issue?

**Comment by tmat** [comment]
likely

**Comment by nguerrera** [comment]
There should probably be a test that catches duplicates.

**Comment by nguerrera** [comment]
I went ahead and submitted small PR so that my subsequent changes don't trigger further conflicts here.
